dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ReturnTypeChangedProblem.java

```diff
@@ -24,15 +26,32 @@
  * actualTypeName}).
  */
 class ReturnTypeChangedProblem extends LinkageProblem {
+  private String actualType;
+
   ReturnTypeChangedProblem(
       ClassFile sourceClass,
       @Nullable ClassFile targetClass,
       MethodSymbol expectedMethodSymbol,
       String actualType) {
     super(
-        "is not found. The expected return type does not match the actual type " + actualType,
+        "is expected to return "
+            + Utility.methodSignatureReturnType(expectedMethodSymbol.getDescriptor())
+            + " but the actual type is "
```

Comment:
"the actual type is" --> but instead returns

Reply:
Nice. Updated.